test/TickMath.t.sol

```diff
@@ -44,6 +49,68 @@ contract TickMathTestTest is Test, JavascriptFfi {
         assertEq(maxTick, MAX_TICK);
     }
 
```

Comment:
Also this comment from @aadams on your previous PR
https://github.com/Uniswap/v4-core/pull/468#discussion_r1486490296